package.json

```diff
@@ -164,11 +164,17 @@
       ],
       "order": 8
     },
+    "cleanAfterBuild": {
+      "title": "Clean after a build.",
```

Comment:
I will push a description with the title fix. Let me know if you don't like the wording :)